packages/cip30/src/BrowserExtension/uiWallet.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  GetBalance,
+  GetChangeAddress,
+  GetRewardAddresses,
+  GetUnusedAddresses,
+  GetUsedAddresses,
+  GetUtxos,
+  SignData,
+  SignTx,
+  SubmitTx,
+  WalletApi
+} from '..';
+import browser from 'webextension-polyfill';
+import { Message } from './Message';
+
+const getUtxos: GetUtxos = (...args) => {
+  const msg: Message = { method: 'getUtxos', arguments: args };
+
+  return browser.runtime.sendMessage(msg);
```

Comment:
Let's add a `sendMessage` function so we can include a log line on the send side